identity-service/src/relay/ethTxRelay.js

```diff
@@ -0,0 +1,185 @@
+const EthereumWallet = require('ethereumjs-wallet')
+const EthereumTx = require('ethereumjs-tx')
+const axios = require('axios')
+const config = require('../config')
+const ethRelayerConfigs = config.get('ethRelayerWallets')
+const { ethWeb3 } = require('../web3')
+
+let ethRelayerWallets = [...ethRelayerConfigs] // will be array of { locked, publicKey, privateKey }
+ethRelayerWallets.forEach(wallet => {
+    wallet.locked = false
+})
+
+async function delay (ms) {
+  return new Promise(resolve => setTimeout(resolve, ms))
+}
+
+// Calculates index into eth relayer addresses
+const getEthRelayerWalletIndex = (walletAddress) => {
+    let walletParsedInteger = parseInt(walletAddress, 16)
+    return walletParsedInteger % ethRelayerWallets.length
+}
+
+// Select from the list of eth relay wallet addresses
+// Return the public key that will be used to relay this address
+const queryEthRelayerWallet = (walletAddress) => {
+    return ethRelayerWallets[getEthRelayerWalletIndex(walletAddress)].publicKey
+}
+
+const getEthRelayerFunds = async (walletPublicKey) => {
+    return ethWeb3.eth.getBalance(walletPublicKey)
+}
+
+const selectEthWallet = async (walletPublicKey) => {
+  // TIMEOUT
+  let resolvedIndex = getEthRelayerWalletIndex(walletPublicKey)
+  while (ethRelayerWallets[index].locked) {
```

Comment:
this delay needs to be moved to the sendEthTransaction function. this function should just check if the wallet is locked. if it is, return null, else, set locked=true and return the wallet